engine/core/src/main/java/org/datacleaner/configuration/RemoteServerConfigurationImpl.java

```diff
@@ -56,5 +87,103 @@ public RemoteServerData getServerConfig(String serverName) {
         return null;
     }
 
+    @Override
+    public RemoteServerState getActualState(String remoteServerName) {
+        if (serverStatusTask == null) {
```

Comment:
I mean, there's a check for `if (serverStatusTask == null)` in `scheduledTask()` (along with a check for the runner), so you don't need to check it here as well, you can just call scheduledTask() directly.

Not that it's much of an issue.